react-search-component/my-app/src/SearchableList.tsx

```diff
@@ -0,0 +1,39 @@
+import { Dispatch, SetStateAction, useState } from 'react';
+
+type Props = {
+  list: string[];
+  value: string;
+  setValue: Dispatch<SetStateAction<string>>;
+};
+
+export function SearchableList({ list }: { list: string[] }) {
+  const [value, setValue] = useState('');
+
+  return (
+    <>
+      <SearchBar list={list} value={value} setValue={setValue} />
+      <MakeList list={list} value={value} setValue={setValue} />
+    </>
+  );
+}
+
+function SearchBar({ value, setValue }: Props) {
```

Comment:
Oh, it looks like you are trying to reuse the `Props` type in all your subcomponents. Don't do that. Declare a type for each one separately. It's very rare that you would have the same props for all of the components, and you want each one carefully typed so that TypeScript can do error checking for you.